c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -580,14 +606,31 @@ mfxStatus MfxC2EncoderComponent::ResetSettings()
     m_mfxVideoParamsConfig.NumExtParam = 0;
     m_mfxVideoParamsConfig.ExtParam = nullptr;
 
-    if (m_encoderType == ENCODER_VP9) {
+    MFX_DEBUG_TRACE__mfxStatus(mfx_res);
+    return mfx_res;
+}
+
+void MfxC2EncoderComponent::AttachExtBuffer()
+{
+    MFX_DEBUG_TRACE_FUNC;
```

Comment:
Whether the h264/h265 different with VP9, is the msdk API requirement?

Reply:
Video Signal Info is applicable to h264 and h265 per codec standard. VP9 bitstream doesn't contain video signal information.